### PR TITLE
Added links to the badges in README.md

### PR DESCRIPTION
**Pull request by mboukhlouf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # ![Zaczero/2Captcha logo](https://github.com/Zaczero/2Captcha/blob/master/resources/2captcha.png)
 
-![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)
-![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)
-![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)
+[![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/releases/latest) [![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)](https://www.nuget.org/packages/2CaptchaAPI/) [![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/blob/master/LICENSE)
```
Comment (Zaczero) [inline README.md]: Is there a reason why everything is in one line? It's hard to read long 1-line code.

```suggestion
[![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/releases/latest)
[![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)](https://www.nuget.org/packages/2CaptchaAPI/)
[![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/blob/master/LICENSE)
```

Comment (mboukhlouf) [reply README.md]: I was using https://stackedit.io/ as a markdown editor. The preview showed the badges on different lines as well.

**Review by Zaczero — APPROVED** [review]

**Comment by Zaczero** [comment]
You can do a similar pull request on my other library for captcha solving service: https://github.com/Zaczero/AntiCaptcha

Obviously, not forcing you to do so. Just saying so. 😃

**Comment by mboukhlouf** [comment]
Absolutely haha. That was my very first pull request.

**Pull request merged**